src/components/FeatureViews/AdminPortal/UnpaidSessionCard.js

```diff
@@ -27,46 +29,54 @@ const UnpaidSessionCard = ({unpaidStudent}) => {
         adminUtils.calculateSessionLength(startTime, endTime)
     );
 
+    const studentID = student.user_id;
+    const courseID = course.course_id;
+
     return (
         <Card className="unpaid-sessions-card">
-            <CardMedia>
-                <Grid
-                    className="unpaid-avatar-container"
-                    container>
-                    <Avatar
-                        className="unpaid-avatar"
-                        style={{
-                            "backgroundColor": stringToColor(student.name),
-                        }}>
-                        {initials(
-                            student.first_name, student.last_name
-                        )}
-                    </Avatar>
-                </Grid>
-            </CardMedia>
-            <CardContent className="unpaid-details">
-                <Typography className="unpaid-student-name">
-                    {student.name}
-                </Typography>
-                <Typography className="unpaid-role-label">
-                    Student
-                </Typography>
-                <Typography>
-                    Payment Status: <span
-                        className="unpaid-status"
-                        style={{
-                            "backgroundColor": adminUtils.statusColor[
-                                unpaidStudent.sessions_left
-                            ],
-                        }}>
-                        {unpaidStudent.sessions_left}
-                    </span>
-                    <br />
-                    ${amtDue}
-                    <br />
-                    {course.title}
-                </Typography>
-            </CardContent>
+            <CardActionArea
+            component = {NavLink}
+            to={`/accounts/students/${studentID}/${courseID}`}
+            >
+                <CardMedia>
+                    <Grid
+                        className="unpaid-avatar-container"
+                        container>
```

Comment:
Why do you need a <Grid container/> for this? shouldn't a div be enough?

Reply:
I think there is a word missing in this comment?

Reply:
I don't understand what this comment is asking for.

Reply:
*Why do you need a container for this? Shouldn't a div be enough?